lib/oli/delivery/evaluation/evaluator.ex

```diff
@@ -9,7 +9,7 @@ defmodule Oli.Delivery.Evaluation.Evaluator do
   evaluation, returns the feedback and a scoring result.
   """
   def evaluate(%Part{} = part, %EvaluationContext{} = context) do
-    case Enum.reduce(part.responses, {context, nil, 0, 0}, &consider_response/2) do
+    case Enum.reduce(part.responses, {context, nil, -1, -1}, &consider_response/2) do
```

Comment:
This needs to remain -1, -1, otherwise "Response-less" activiites (e.g. ones that use client-side evaluation) that haven't been submitted will display "0 out of -1" points.

Reply:
The trouble with the scores starting at `0` is that the `evaluate` logic only uses matching responses that have a **greater** score, meaning the way it worked before is that if the student's answer is incorrect, it updates the responses with the first matching targeted feedback. If we change the logic to allow responses with scores **greater than or equal to** the current score, it will take the last catch-all response instead.

There's probably a way to change it so it takes the first response that is at least the current score, I'll think about it after the multi input is wrapped up.